internal/utils/utils.go

```diff
@@ -241,3 +242,16 @@ func GetPortFromDiff(port string, diff int) string {
 	GetLogInstance().Error("error on parsing port.")
 	return ""
 }
+
+// ReadStringFromFile reads string from a file.
+func ReadStringFromFile(fileName string) string {
+	data, err := ioutil.ReadFile(fileName)
+	check(err)
+	return string(data)
+}
+
+func check(err error) {
```

Comment:
don't panic

Reply:
+1.  In fact, let's not have this function: Not only it is unused, but panicking on error should not be made into an easy-to-use utility function.